code/stable/gamephysics/SRS/GamePhysics_SRS.tex

```diff
@@ -986,7 +986,7 @@ \subsubsection{Data Definitions}
         
 \\ \midrule \\
 Equation & \begin{displaymath}
-           KE=\frac{m \mathbf{v}^{2}}{2}
+           KE=\frac{m \mathbf{v}^{2.0}}{2.0}
```

Comment:
That `2.0` in the denominator might also be exact, but I'm not sure. The meaning might be "cut in half" rather than "divide by 2.0".